socorro/processor/breakpad_transform_rules.py

```diff
@@ -459,3 +491,153 @@ def _predicate(
         return 'create_dump_lookup' in raw_crash
 
 
+#==============================================================================
+class BreakpadStackwalkerRule2015(ExternalProcessRule):
+
+    required_config = Namespace()
+    required_config.add_option(
+        name='symbols_url',
+        doc='url of the symbol server',
+        default="https://localhost"
```

Comment:
Doesn't this need one of those [likely_to_be_changed=True](https://github.com/mozilla/configman/blob/master/configman/option.py#L33) things.